Makefile

```diff
@@ -0,0 +1,32 @@
+DEFAULT_VERSION = 'latest'
+DEFAULT_COMMAND = 'generate'
+TEST_FILEPATH = 'test/asyncapi.yml'
+DEFAULT_TEMPLATE = '@asyncapi/markdown-template@0.10.0'
+DEFAULT_LANGUAGE = ''
+DEFAULT_OUTPUT = 'output'
+DEFAULT_PARAMETERS = ''
+DEFAULT_CUSTOM_COMMANDS = ''
+CUSTOM_COMMANDS = 'validate test/asyncapi.yml'
+
+run:
+	@bash ./entrypoint.sh $(DEFAULT_VERSION) $(DEFAULT_COMMAND) $(TEST_FILEPATH) $(DEFAULT_TEMPLATE) $(DEFAULT_LANGUAGE) $(DEFAULT_OUTPUT) $(DEFAULT_PARAMETERS) $(DEFAULT_CUSTOM_COMMANDS) 
+
+test: test-1 test-2 test-3 test-4 test-5
+
+test-1:
```

Comment:
can you name them in more transparent way, like `generate-`something ? also add description to each test, like a comment you know that provides human-readable name of test, and best would be what AsyncAPI CLI command is tested - as you know with these default values it is not really readable

Reply:
Okay

Reply:
Done